xwiki-pro-macros-ui/src/main/resources/XWiki/Macros/Team.xml

```diff
@@ -851,6 +851,15 @@
     #end
   #end
   {{html wiki=false clean=false}}
+  ##Starting with XWiki version 13.10.4 and 14.1-rc-1, img tags have a height: auto set and this breaks the macro fixed size feature. For this, a specific size is added
+  ## https://jira.xwiki.org/browse/XWIKI-19432
```

Comment:
It is better to add the actual title of the jira issue. Like this, other devs don't need an extra click to know what the issue it's actual about and the comment is easier to read. E.g. https://github.com/xwikisas/application-licensing/blob/d168fb88fc0d121bf95e769ea21c55c00bebe5a6/application-licensing-licensor/application-licensing-licensor-api/src/main/java/com/xwiki/licensing/internal/upgrades/LicensingSchedulerListener.java#L51

Reply:
Also, did you also tested the view on mobile? Asking because of https://jira.xwiki.org/browse/XWIKI-19432